kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -193,8 +193,19 @@
               routeBase = '/topics/t';
```

Comment:
@jonboiser's suggestions about using `is_leaf` and a route object apply here too.